stable/traefik/values.yaml

```diff
@@ -1,6 +1,8 @@
 ## Default values for Traefik
 image: traefik
 imageTag: 1.4.5
+## can switch the deploy type to Deployment if you prefer
+deployType: Daemonset
```

Comment:
Shouldn't Daemonset be [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)? We should keep consistency with kubernetes' resources names.

Also, since the last default behavior was **Deployment** is it safe to silently switch to **DaemonSets**? Isn't this a breaking change? If so, chart version should be incremented by its MAJOR value instead of MINOR value as described by [SemVer2](https://semver.org/).
> Given a version number MAJOR.MINOR.PATCH, increment the:
> MINOR version when you add functionality in a **backwards-compatible** manner

Reply:
in some way, it's a breaking change, see https://github.com/containous/traefik/issues/2633,
i will change the default deployType to Deployment, but i think bump MINIOR version is ok here